File: core/src/test/java/examples/java/noreceiver/doublecomparison/Correct0.java
package examples.java.noreceiver.doublecomparison;

public class Correct0 {
  public double sum(double[] values) {
    double sum = 0.0;
    for (int i = values.length - 1; i >= 0; i--) {
      sum += values[i];
    }
    return sum;
  }
}

File: src/test/java/examples/java/noreceiver/arrayreturn/Correct0.kt
package examples.java.noreceiver.arrayreturn

class Correct0 {
    fun biggerTwo(first: IntArray?, second: IntArray?): IntArray {
        val aSum = first!![0] + first[1]
        val bSum = second!![0] + second[1]
        return if (bSum > aSum) {
            second
        } else {
            first
        }
    }
}